### PR TITLE
JAMES-2884 Improve JMAP annotated documentation

### PR DESCRIPTION
**Pull request by chibenwa**
- 1. Point to it from server documentation
 - 2. List tested clients
 - 3. Document authentication

### CODE REVIEW

File: docs/modules/servers/pages/distributed/configure/jmap.adoc [hunk 1]
```diff
@@ -77,3 +77,22 @@ This token can then be passed as `Bearer` of the `Authorization` header :
...
+
+Users are invited to read these limitations before using actively the JMAP RFC-8621 implementation, and should ensure their
+client applications only uses supported operations.
+
+Contributions enhancing support are furthermore welcomed.
+
+The list of tested JMAP clients are:
+
+ - [OpenPaaS](https://open-paas.org/) is actively using the draft version of the JMAP implementation. Migration to
+ RFC-8621 is planed.
```
Comment (LanKhuat) [inline docs/modules/servers/pages/distributed/configure/jmap.adoc:96]: s/planed/planned

File: server/protocols/jmap-rfc-8621/doc/specs/spec/authentication.mdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Authentication
+
+JMAP RFC-8620 deliberately do not address authentication concerns, and only assume authenticated requests are handled.
+
+Discovery of available authentication mechanism is not part of the JMAP specification.
+
+This document summarizes available authentication mechanism on top of JMAP RFC-8621  implementation as part of the James
```
Comment (LanKhuat) [inline server/protocols/jmap-rfc-8621/doc/specs/spec/authentication.mdown:7]: extra space

File: src/site/xdoc/server/config-jmap.xml [hunk 3]
```diff
@@ -104,6 +104,25 @@
...
+                <p>Some methods / types are not yet implemented, some implementations are naive, and the PUSH is not supported yet.</p>
+
+                <p>Users are invited to read these limitations before using actively the JMAP RFC-8621 implementation, and should ensure their
+                client applications only uses supported operations.</p>
+
+                <p>Contributions enhancing support are furthermore welcomed.</p>
+
+                <ul>The list of tested JMAP clients are:
+                    <li><a href="https://open-paas.org/">OpenPaaS</a> is actively using the draft version of the JMAP implementation. Migration to
+                    RFC-8621 is planed.</li>
```
Comment (LanKhuat) [inline src/site/xdoc/server/config-jmap.xml:121]: s/planed/planned

**Review by LanKhuat — APPROVED** [review]

File: server/protocols/jmap-rfc-8621/doc/specs/spec/authentication.mdown [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Authentication
+
+JMAP RFC-8620 deliberately do not address authentication concerns, and only assume authenticated requests are handled.
```
Comment (Arsnael) [inline server/protocols/jmap-rfc-8621/doc/specs/spec/authentication.mdown:3]:
```suggestion
JMAP RFC-8620 deliberately does not address authentication concerns, and only assumes authenticated requests are handled.
```

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**